examples/numalogic-simple-pipeline/ml_steps/README.md

```diff
@@ -0,0 +1,52 @@
+# Simple Numalogic Pipeline 
+
+## Running Example: Simple Numalogic Pipeline
+1. Build the docker image, and push
+```
+docker build -t simple-numalogic-pipeline:v1 . && k3d image import docker.io/library/simple-numalogic-pipeline:v1
+```
+2. Apply the pipeline
+```
+kubectl apply -f simple-numalogic-pipeline.yaml
+```
+
```

Comment:
you might want to talk about `kubectl get pods` and wait for pipeline to be ready.

Before that, the developer should have `numaflow` running.